slack-api-client/src/test/java/test_with_remote_apis/methods/channels_Test.java

```diff
@@ -94,17 +94,27 @@ public void channels_threading() throws IOException, SlackApiException {
         assertThat(firstMessageCreation.getError(), is(nullValue()));
         assertThat(firstMessageCreation.isOk(), is(true));
 
+        {
+            ChatPostMessageResponse reply1 = slack.methods().chatPostMessage(req -> req
+                            .channel(randomChannelId)
+                            .token(userToken)
+                            .asUser(false)
+                            .text("replied")
+                            .iconEmoji(":smile:")
+                            .threadTs(firstMessageCreation.getTs())
+            );
+            assertThat(reply1.getError(), is("invalid_arguments"));
+            assertThat(reply1.getDeprecatedArgument(), is("as_user"));
```

Comment:
This error response indicates that `as_user` won't be available for any cases. If a Slack app needs to post a message on behalf of a user, the app can use the person's user token, not a bot token.